nondex-maven-plugin/src/main/java/edu/illinois/nondex/plugin/CleanSurefireExecution.java

```diff
@@ -184,4 +186,15 @@ protected Xpp3Dom makeNode(String nodeName, String value) {
         node.setValue(value);
         return node;
     }
+
+    protected static String sanitizeAndRemoveEnvironmentVars(String toSanitize) {
```

Comment:
No. The tests demonstrate what it does. It removes any string of the form ${something} which is how properties are referenced typically in maven. This is somewhat fishy that I can get unresolved variables from Maven, but I need to make my code robust.